versioned_docs/version-8.7/components/modeler/desktop-modeler/model-your-first-diagram.md

```diff
@@ -20,7 +20,7 @@ The BPMN diagram opens with a start event. [Events](/components/modeler/bpmn/eve
 
 ![elements](./img/elements.png)
 
-Above, you can see that elements that support different types can be reconfigured by clicking on the corresponding icon. In this case, the task can be converted to a [service task](../bpmn/service-tasks/service-tasks.md), for example, by clicking on the task and the wrench-shaped icon.
```

Comment:
I need to confirm if this is still the case in Desktop Modeler.

Reply:
Still, both icons are called `Change element`, so it does not matter whether the icon depicts a wrench or another hand tool.
![image](https://github.com/user-attachments/assets/b6de37fd-c959-4cc7-9b29-517445e604c6)